terraform/variables.tf

```diff
@@ -66,14 +66,17 @@ variable "cyhy_archive_bucket_name" {
   default = "ncats-cyhy-archive"
 }
 
-variable "cyhy_portscan_elastic_ips" {
-  type = "list"
-  description = "The IP addresses of the elastic IPs to be assigned to the CyHy portscan instances.  These values are only used in a production workspace, and in that case the number of elements in the list should equal the number of portscan (nmap) instances being created."
-  default = []
+variable "cyhy_elastic_ip_cidr_block" {
+  description = "The CIDR block of elastic addresses available for use by CyHy scanner instances."
+  default = ""
 }
 
-variable "cyhy_vulnscan_elastic_ips" {
-  type = "list"
-  description = "The IP addresses of the elastic IPs to be assigned to the CyHy vulnscan instances.  These values are only used in a production workspace, and in that case the number of elements in the list should equal the number of vulnscan (Nessus) instances being created."
-  default = []
+variable "cyhy_portscan_first_elastic_ip_offset" {
+  description = "The offset of the address (from the start of the elastic IP CIDR block) to be assigned to the *first* CyHy portscan instance.  For example, if the CIDR block is 192.168.1.0/24 and the offset is set to 10, the first portscan address used will be 192.168.1.10.  This is only used in production workspaces.  Each additional portscan instance will get the next consecutive address in the block.  NOTE: This will only work as intended when a contiguous CIDR block of EIP addresses is available."
+  default = 0
+}
+
+variable "cyhy_vulnscan_first_elastic_ip_offset" {
+  description = "The The offset of the address (from the start of the elastic IP CIDR block) to be assigned to the *first* CyHy vulnscan instance.  For example, if the CIDR block is 192.168.1.0/24 and the offset is set to 10, the first vulnscan address used will be 192.168.1.10.  This is only used in production workspaces.  Each additional vulnscan instance will get the next consecutive address in the block.  NOTE: This will only work as intended when a contiguous CIDR block of EIP addresses is available."
```

Comment:
Duplicate "The".

Reply:
Resolved in [this commit](https://github.com/dhs-ncats/cyhy_amis/pull/146/commits/bd4335618c1a62fc7456a4f1b836620b10bb933f)